README.md

```diff
@@ -43,13 +43,17 @@ Add this script in a file `script.js`.
 
 ```js
 const { openBrowser, closeBrowser, click, goto, video } = require('taiko');
+const { slowdownRecording } = require('./src');
```

Comment:
**nit:** this won't be necessary since you can call `video.slowdownRecording` right?